pages/mesosphere/dcos/services/edge-lb/1.5/how-to-tasks/scale-up/index.md

```diff
@@ -0,0 +1,187 @@
+        }
+      }
+    }
+    ```
+
+    In this sample baseline configuration, the `count` variable is set to 1. With this setting, the Edge-LB pool uses one load balancer instance from one of the public agents to distribute incoming traffic from all available public agent nodes.
+
+1. Modify the sample Edge-LB pool configuration file, ```scaleup-nginx-lb.json```, to scale up the number of Edge-LB pool instances by changing the `count` setting from 1 to 3.
+
+    ```json
+    {
+      "apiVersion": "V2",
+      "name": "scaleup-nginx-lb",
+      "count": 3,
+      "haproxy": {
+        "frontends": [
+        {
+          "bindPort": 15002,
+          "protocol": "HTTP",
+          "linkBackend": {
+            "defaultBackend": "nginx-backend"
+          }
+        }
+      ],
+        "backends": [
+        {
+          "name": "nginx-backend",
+          "protocol": "HTTP",
+          "services": [
+            {
+            "marathon": {
+              "serviceID": "/nginx"
+            },
+            "endpoint": {
+              "portName": "nginx-port"
+            }
+          }
+        ]
+        }
+      ],
+        "stats": {
+          "bindPort": 0
+        }
+      }
+    }
+    ```
+
+1. Update the Edge-LB pool to use the new configuration settings:
+
+    ```bash
+    dcos edgelb update scaleup-nginx-lb.json
+    ```
+
+1. Verify the change to the number of load balancer instances has been deployed successfully:
+
+    ```
+    dcos edgelb status scaleup-nginx-lb
+    ```
+
+    You should see two pools listed in the output for this command. You can also see the Edge-LB endpoints with the `dcos edgelb endpoint` command. In the command output, the `count` column displays the number of load balancer instances you have specified in the Edge-LB pool configuration file.
+    
+    To view the output from the endpoints subcommand:
+
+    ```
+    dcos edgelb endpoints scaleup-nginx-lb
+    ```
+
+    You can only scale up to the total number of public agents you have available in the DC/OS cluster. For example, if you have deployed five public agent nodes in your DC/OS cluster, you can scale up to a maximum of five Edge-LB load balancer instances. If you change the `count` setting to a number greater than the number of public agents you have deployed - for example, by setting the `count` to 6 - only the maximum of five Edge-LB load balancer instances are available to handle the traffic from the five available public agents.
+
+# Scale down Edge-LB pool instances
+To scale down Edge-LB pool instances, you can change the `count` setting to a lower number. For example, if the previous count in the Edge-LB pool configuration file was 5, you can scale down the number of instances available by changing the `count` setting to 2.
+
+To scale down the number of Edge-LB load balancer instances:
+1. Open and modify the Edge-LB configuration file:
+
+    ```json
+    {
+      "apiVersion": "V2",
+      "name": "scaledown-nginx-lb",
```

Comment:
This `name` is different than the pool name above, therefore it would not scale down anything, but instead create a new pool.

Reply:
created separate ticket

Reply:
Created a separate ticket.